Sources/Apollo/GraphQLHTTPRequestError.swift

```diff
@@ -1,10 +1,13 @@
 /// An error which has occurred during the serialization of a request.
 public enum GraphQLHTTPRequestError: Error, LocalizedError {
+  case cancelledByDeveloper
   case serializedBodyMessageError
   case serializedQueryParamsMessageError
   
   public var errorDescription: String? {
     switch self {
+    case .cancelledByDeveloper:
+      return "The request was cancelled by the developer using the HTTPNetworkTransportPreflightDelegate."
```

Comment:
Not sure if this should mention "the developer", because then you could attribute almost all implementation to "the developer".

Maybe something like `cancelledByPreflight` with "The request was cancelled by the the HTTPNetworkTransportPreflightDelegate" is more suitable?

Reply:
Fair point